### DISCUSSION TITLE
Is the latest release (obs-multi-rtmp 0.5.0.4) working with Ubuntu 22.04 LTS?

### DISCUSSION THREAD

**Discussion by bStyler** [sorayuki/obs-multi-rtmp#377] [General]
Hey @sorayuki,

I'm still stuck with OBS 27 because I failed to install the multi-rtmp plugin in OBS 28/29. I'd love to upgrade to OBS 30 but I am concerned that I won't be able to build the plugin.

Can you please advise or did anyone else try and make it work? I'd appreciate any advice.

Thank you!

**Comment by sorayuki** [comment]
![a](https://github.com/sorayuki/obs-multi-rtmp/assets/16860691/744510c5-f809-4c6d-8975-318f685687bc)
I don't know, but it works with OBS 30 in Fedora 39

**Reply by bStyler** [reply]
Nice! And what are the commands to get it installed?

**Reply by sorayuki** [reply]
I'm now dealing with this issue.

https://github.com/sorayuki/obs-multi-rtmp/issues/375

The path structure seems to be not usable without rearrange manually

**Reply by sorayuki** [reply]
try this
https://github.com/sorayuki/obs-multi-rtmp/releases/tag/0.5.0.4-flatpak

**Reply by bStyler** [reply]
Thanks! Can you clarify how do I use the flatpak or what commands should I run to have it installed?

**Reply by sorayuki** [reply]
Extract .tar.xz to correct place according to the table in release page
